### DISCUSSION TITLE
Error on opt-322-lotresult

### DISCUSSION THREAD

**Discussion by user6330** [OP-TED/eForms-SDK#702] [Q&A] [answered]
I have this xml

```
<ContractAwardNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:ContractAwardNotice-2"
                     ....">
   <ext:UBLExtensions>
      <ext:UBLExtension>
         <ext:ExtensionContent>
            <efext:EformsExtension>
               <efac:NoticeResult>
                  <efac:LotResult>
                     <cbc:ID>RES-0001</cbc:ID>
                     <cbc:TenderResultCode listName="winner-selection-status">clos-nw</cbc:TenderResultCode>
                  </efac:LotResult>
.....
```

but the xsd gave me this error, 
 "**Any LotResult Technical Identifier (OPT-322-LotResult) must differ from the following LotResult Technical Identifier (OPT-322-LotResult)**"

what's the problem here?

**Comment by rouschr** [comment]
Hello, 
I do not have your full XML, but it looks like you could have several cbc:ID having the same value "RES-0001".
Can you check your XML?

**Comment by user6330** [comment]
hello,

For privacy content I can't upload the XML but this is the confirm that RES-0001 is unique.

![image](https://github.com/OP-TED/eForms-SDK/assets/127392893/e6c1be2e-6495-4a88-b4fd-7e3115a50c4a)

before giving me the error I needed to do in order:


1. set value to this field **BT-13713-LotResult** because via xsd,the tag _<efac:LotResult>_ needed _<efac:TenderLot>_ tag.
![image](https://github.com/OP-TED/eForms-SDK/assets/127392893/9c6b9f56-7456-47e5-8fd6-0136980aa8c6)
2. changed **BT-137-Lot** from LOT-0001 into LOT-0002 for this error 'purpose lot identifier (bt-137) value must be unique'.

Maybe this is connected?

**Comment by rouschr** [comment, marked as answer]
Please see this with the helpdesk: https://simap.ted.europa.eu/contact
